snapshots/output/basic-project/lib/relationships.dart

```diff
@@ -1,9 +1,16 @@
   abstract class Mammal {
 // definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/
 //               ^^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Mammal#
-    String get hierarchy;
+    String get someGetter;
 //  ^^^^^^ reference scip-dart pub dart:core 2.19.0 dart:core/`string.dart`/String#
-//             ^^^^^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Mammal#hierarchy.
+//             ^^^^^^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Mammal#`<get>someGetter`.
+    set someSetter(String v);
+//      ^^^^^^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Mammal#`<set>someSetter`.
+//                 ^^^^^^ reference scip-dart pub dart:core 2.19.0 dart:core/`string.dart`/String#
+//                        ^ definition local 0
+    String field = '';
+//  ^^^^^^ reference scip-dart pub dart:core 2.19.0 dart:core/`string.dart`/String#
+//         ^^^^^ definition scip-dart pub dart_test 1.0.0 lib/`relationships.dart`/Mammal#field.
```

Comment:
`Mammal` has sub-classes extending it, but it is, itself, at the top level

All fields, methods, getters and setters, should not contain references, like is shown here